### PR TITLE
re-added colors

### PR DESCRIPTION
**Pull request by IGBC**
Fixing #909

### CODE REVIEW

**Review by nobbele — APPROVED** [review]

**Comment by IGBC** [comment]
@icefoxen broke the build in 798dd3e793f94312a8cb695258ce8b586a460eac a couple of hours ago. I guess I need to wait for the build to be fixed before we can merge.

**Pull request merged**
